lib/ember-orbit/record-array-manager.js

```diff
@@ -35,11 +35,11 @@ var RecordArrayManager = Ember.Object.extend({
    @private
    */
   _processChanges: function() {
-    forEach(this.changes, function(change) {
-      this._processChange(change.record, change.operation);
-    }, this);
+    var change;
```

Comment:
I figured one less unnecessary closure/iterator in the "hot path" of property change might be useful, all tests still pass.  Happy to undo/do alone.